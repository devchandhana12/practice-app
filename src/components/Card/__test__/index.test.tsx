import React from "react";
import CardComp from "..";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";

afterEach(cleanup);
it("should match snapshot", () => {
  const { asFragment } = render(
    <CardComp>
      <p>hello</p>
    </CardComp>
  );
  expect(asFragment()).toMatchSnapshot();
});
