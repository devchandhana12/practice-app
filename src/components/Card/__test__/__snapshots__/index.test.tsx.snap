// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class=" shadow-lg text-white  card"
    style="background-color: rgb(39, 40, 41); display: flex; flex-direction: column; align-items: center;"
  >
    <div
      class="card-body"
    >
      <p>
        hello
      </p>
    </div>
  </div>
</DocumentFragment>
`;
